_posts/en/newsletters/2019-02-19-newsletter.md

```diff
@@ -0,0 +1,252 @@
+    preventing any miscommunications or misunderstandings from ending in
+    loss of funds.
+
+    Renewed discussion [last week][nick output tagging] and this week
+    saw analysis of the impact this would have on proposed layer two
+    protocols such as Eltoo and [Channel Factories][].  Although the
+    proposal obviously increases complexity, the general opinion seems
+    to be that it doesn't fundamentally increase the cost or reduce the
+    effectiveness of the described proposals, although it could make
+    them a bit less private (because you'd communicate that noinput was
+    an option even in cases where you don't end up using
+    noinput---likely giving away that a second-layer protocol was in
+    use).
+
+- **Bitcoin Core preliminary hardware wallet support:** after months of
+  incremental improvements, this week saw the merge of the final set of
+  PRs needed for Bitcoin Core's master development branch to support
+  receiving and sending transactions in conjunction with a hardware
+  wallet via the [Hardware Wallet Interaction][HWI] (HWI) tool.  HWI is
+  not yet distributed with Bitcoin Core (although it is part of the
+  overall project) and is currently only accessible from the command
+  line, limiting the size of its userbase, but it provides a solid
+  foundation upon which to build tools that can make it easy to use an
+  external keystore with Bitcoin Core's native wallet and full
+  verification node.  (Note, alternatively, it's already possible to
+  connect a hardware wallet to an Electrum wallet connected to your full
+  node using [Electrum Personal Server][].)
+
+    Organizations using advanced security techniques such as Hardware
+    Security Modules (HSMs), cold wallets, and multisig may want to
+    investigate the design of HWI and how it interacts with Bitcoin Core
+    using [output script descriptors][descriptor] and [BIP174][] PSBTs.
+    These next-generation encodings of key and transaction data (and
+    metadata), along with other advances such as the [miniscript][]
+    policy language, make it easier than ever to build and operate
+    secure bitcoin storage solutions that interact with a full node for
+    verification.
+
+- **Bitcoin Core freeze week:** as [scheduled][Bitcoin Core #14438],
+  the project has stopped accepting features for the upcoming release of
+  major version 0.18.  As often happens, this was preceded by a week or
+  so of active last-minute review and merging of new features, which is
+  reflected in the long list of changes in this week's *notable changes*
+  section below.  The next two weeks will focus on developer testing and
+  bug fixes, followed by the issuance of Release Candidates (RCs) for
+  user testing.  The RC cycle for a major release usually lasts two to
+  four weeks before a final release.
+
+    Related, the project prefers to merge major new features early in a
+    new development cycle so that they get as much post-ACK developer
```

Comment:
nit: post-ACK developer testing is a bit jargony. Perhaps "as much soak time" would be more widely understood.

Reply:
Agree that post-ACK is jargony, but I've never heard the term "soak time" myself, so I'm changing this to the much more boring but non-jargon or colloquial "additional developer testing".